src/NuGet.Clients/PackageManagement.VisualStudio/Strings.resx

```diff
@@ -205,7 +205,7 @@
     <comment>{0} is the full path to the project.</comment>
   </data>
   <data name="ProjectCouldNotBeCastedToBuildPropertyStorage" xml:space="preserve">
-    <value>The project '{0}' could not be casted to a build property storage interace, which is required to get MSBuild properties inside Visual Studio.</value>
+    <value>The project '{0}' could not be casted to a build property storage interface, which is required to get MSBuild properties inside Visual Studio.</value>
```

Comment:
I'm not sure this message will make sense to a user. It should say the project is not supported. And if needed name the exact interface that is required.